server/configuration.go

```diff
@@ -220,7 +220,7 @@ func (p *Plugin) ensureDemoUser(configuration *configuration) (string, error) {
 	if user == nil {
 		user, err = p.API.CreateUser(&model.User{
 			Username:  configuration.Username,
-			Password:  "password",
+			Password:  "P@ssw0rd!!",
```

Comment:
Why is this change needed? :question:

Reply:
Whoops.  I was testing and on a branch there was a 10 character + special characters limit.  Changing this back